machine/v1beta1/types_awsprovider.go

```diff
@@ -428,7 +441,7 @@ type HostPlacement struct {
 	Affinity *HostAffinity `json:"affinity,omitempty"`
 
 	// dedicatedHost specifies the exact host that an instance should be restarted on if stopped.
-	// dedicatedHost is required when 'affinity' is set to DedicatedHost, and forbidden otherwise.
+	// dedicatedHost is required when 'affinity' is set to DedicatedHost, and optional otherwise.
```

Comment:
What does it mean if I set `affinity` to `AnyAvailable` and set `dedicatedHost`? Is that a valid configuration?